### PR TITLE
docs: memberships privacy

### PR DESCRIPTION
**Pull request by loks0n**
Adds memberships privacy documentation to 3 pages:
- Auth > Security page
- Platform > Security page
- Auth > Teams concept page

### CODE REVIEW

File: src/routes/docs/products/auth/teams/+page.markdoc [hunk 1]
```diff
@@ -215,4 +215,19 @@ individual roles in the team using the `Role.team(<TEAM_ID>, [<ROLE_1>, <ROLE_2>
 
 {% arrow_link href="/docs/advanced/platform/permissions" %}
 Learn more about permissions
-{% /arrow_link %}
\ No newline at end of file
+{% /arrow_link %}
+
+# Memberships privacy {% #memberships-privacy %}
+
+Hide specific membership details from other users by marking them as private. To set this up, go to **Auth** > **Security** > **Memberships privacy**
```
Comment (eldadfux) [inline src/routes/docs/products/auth/teams/+page.markdoc:222]:
```suggestion
In certain use cases, your app may not need to share members’ personal information with others. You can safeguard privacy by marking specific membership details as private. To configure this setting, navigate to **Auth** > **Security** > **Memberships privacy**
```

File: src/routes/docs/products/auth/security/+page.markdoc [hunk 2]
```diff
@@ -40,3 +40,18 @@ These permissions are enforced for client SDKs and server SDKs when using JWT, b
 Creating and using mock phone numbers allows users to test SMS authentication without needing an actual phone number. This can be useful for testing edge cases where a user doesn't have a phone number but needs to sign in to your application using SMS. 
 
 To create a mock phone number, navigate to **Auth** > **Security** > Mock Phone Numbers. After defining a mock phone number, you need to define a specific OTP code that will be used for SMS sign-in instead of the SMS secret code sent to a real phone number.
+
+# Memberships privacy {% #memberships-privacy %}
+
+Hide specific membership details from other users by marking them as private. To set this up, go to **Auth** > **Security** > **Memberships privacy**
```
Comment (eldadfux) [inline src/routes/docs/products/auth/security/+page.markdoc:46]:
```suggestion
In certain use cases, your app may not need to share members’ personal information with others. You can safeguard privacy by marking specific membership details as private. To configure this setting, navigate to **Auth** > **Security** > **Memberships privacy**
```

File: src/partials/auth-security.md [hunk 3]
```diff
@@ -34,3 +34,18 @@ Enable email alerts for your users so that whenever another session is created f
 You won't receive notifications when logging in using [Magic URL](/docs/products/auth/magic-url), [Email OTP](/docs/products/auth/email-otp), or [OAuth2](/docs/products/auth/oauth2) since these authentication methods already verify user access to their systems, establishing the authentication's legitimacy.
 
 To toggle session alerts, navigate to **Auth** > **Security** > **Session alerts**.
+
+# Memberships privacy {% #memberships-privacy %}
+
+Hide specific membership details from other users by marking them as private. To set this up, go to **Auth** > **Security** > **Memberships privacy**
```
Comment (eldadfux) [inline src/partials/auth-security.md:40]:
```suggestion
In certain use cases, your app may not need to share members’ personal information with others. You can safeguard privacy by marking specific membership details as private. To configure this setting, navigate to **Auth** > **Security** > **Memberships privacy**
```

File: src/partials/auth-security.md [hunk 4]
```diff
@@ -34,3 +34,18 @@ Enable email alerts for your users so that whenever another session is created f
...
+
+These details can be made private:
+
+- `userName` - The member's name
+- `userEmail` - The member's email address
+- `mfa` - Whether the member has enabled multi-factor authentication
+
+{% info title="Scale plan feature" %}
+Memberships privacy is available on the Appwrite Cloud scale plan.
+{% /info %}
```
Comment (eldadfux) [inline src/partials/auth-security.md:50]:
```suggestion
```

We don't need this anymore since we're going to make this feature available to all plans.

File: src/routes/docs/products/auth/security/+page.markdoc [hunk 5]
```diff
@@ -40,3 +40,18 @@ These permissions are enforced for client SDKs and server SDKs when using JWT, b
...
+
+These details can be made private:
+
+- `userName` - The member's name
+- `userEmail` - The member's email address
+- `mfa` - Whether the member has enabled multi-factor authentication
+
+{% info title="Scale plan feature" %}
+Memberships privacy is available on the Appwrite Cloud scale plan.
+{% /info %}
```
Comment (eldadfux) [inline src/routes/docs/products/auth/security/+page.markdoc:56]:
```suggestion
```

We don't need this anymore since we're going to make this feature available to all plans.

File: src/routes/docs/products/auth/teams/+page.markdoc [hunk 6]
```diff
@@ -215,4 +215,19 @@ individual roles in the team using the `Role.team(<TEAM_ID>, [<ROLE_1>, <ROLE_2>
...
+
+These details can be made private:
+
+- `userName` - The member's name
+- `userEmail` - The member's email address
+- `mfa` - Whether the member has enabled multi-factor authentication
+
+{% info title="Scale plan feature" %}
+Memberships privacy is available on the Appwrite Cloud scale plan.
+{% /info %}
```
Comment (eldadfux) [inline src/routes/docs/products/auth/teams/+page.markdoc:232]:
```suggestion
```

We don't need this anymore since we're going to make this feature available to all plans.

**Review by eldadfux — DISMISSED** [review]

**Review by eldadfux — APPROVED** [review]
